src/index.html

```diff
@@ -5,8 +5,288 @@
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <title>Catalog</title>
     <link rel="stylesheet" href="./styles/main.scss">
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&display=swap" rel="stylesheet">
   </head>
   <body>
-    <h1>Catalog</h1>
+    <header class="navigation">
+      <a href="#" class="navigation__logo">
+        <img
+          src="images/logo.png"
+          class="navigation__image"
+          alt="Moyo-Logo"
+        >
+      </a>
+
+      <nav class="navigation__navbar navbar">
+        <ul class="navbar__list">
+          <li class="navbar__item">
+            <a href="#" class="navbar__link is-active">
+              Apple
+            </a>
+          </li>
+          <li class="navbar__item">
+            <a href="#" class="navbar__link">
+              Samsung
+            </a>
+          </li>
+          <li class="navbar__item">
+            <a href="#" class="navbar__link">
+              Смартфоны и телефоны
+            </a>
+          </li>
+          <li class="navbar__item">
+            <a
+              href="#"
+              class="navbar__link"
+              data-qa="nav-hover"
+            >
+              Ноутбуки и компьютеры
+            </a>
+          </li>
+          <li class="navbar__item">
+            <a href="#" class="navbar__link">
+              Гаджеты
+            </a>
+          </li>
+          <li class="navbar__item">
+            <a href="#" class="navbar__link">
+              Планшеты
+            </a>
+          </li>
+          <li class="navbar__item">
+            <a href="#" class="navbar__link">
+              ФОТО
+            </a>
+          </li>
+          <li class="navbar__item">
+            <a href="#" class="navbar__link">
+              ВИДЕО
+            </a>
+          </li>
+        </ul>
+      </nav>
+    </header>
+
+    <main class="container container--wrapper">
```

Comment:
```suggestion
    <main class="container page__container">
```
У тебя остался еще один модификатор с margin-ом. Добавь в **<body>** класс **page** (сделаем его блоком) и сделаем микс блоков **page__container** для **<main>**. К миксу примени стили которые были у модификатора.